### DISCUSSION TITLE
npm ERR! 404 Not Found - GET https://registry.npmjs.org/perspective-js - Not found

### DISCUSSION THREAD

**Discussion by aishamal** [perspective-dev/perspective#2231] [Q&A]
Bug Report
While creating a table and updating the data with a file in realtime

Steps to Reproduce:
 1. https://codesandbox.io/s/react-elcg7i?file 
 2. Clone this code 

 Expected Result:
On doing npm start a data table should get created

Actual Result:
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! perspective@0.1.0 start: `NODE_ENV=production node_modules/react-scripts/bin/react-scripts.js start`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the perspective@0.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/aisha/.npm/_logs/2023-05-21T19_23_16_828Z-debug.log

 Environment:
 "dependencies": {
    "@finos/perspective": "^2.1.1",
    "@finos/perspective-viewer": "^2.1.2",
    "@finos/perspective-viewer-d3fc": "^2.1.2",
    "@finos/perspective-viewer-datagrid": "^2.1.2",
    "@finos/perspective-webpack-plugin": "^2.1.1",
    "@testing-library/jest-dom": "^5.16.5",
    "@testing-library/react": "^13.4.0",
    "@testing-library/user-event": "^13.5.0",
    "perspective-js": "^1.0.3",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-scripts": "^5.0.1",
    "web-vitals": "^2.1.4"

 Additional Context:

<img width="1440" alt="Screenshot 2023-05-22 at 12 53 44 AM" src="https://github.com/finos/perspective/assets/123355544/2eb5461c-b66b-4637-92fb-510214ddd0f7">
<img width="1440" alt="Screenshot 2023-05-22 at 12 54 39 AM" src="https://github.com/finos/perspective/assets/123355544/67a93ef0-351a-4b67-889a-05bff1b35afa">

**Comment by texodus** [comment]
@aishamal The code in your repro sandbox and screenshot do not match, and the repro sandbox works fine.  Please provide the full repro, we can't debug a screenshot.

(PS and the issue title is in neither ...)
